recipes/salt/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2015.8.7" %}
+
+package:
+  name: salt
+  version: {{ version }}
+
+source:
+  git_tag: v{{ version }}
+  git_url: https://github.com/saltstack/salt.git
```

Comment:
@danielfrg you can use these 3 lines instead of the above:

```yaml
fn: salt-{{ version }}.tar.gz
url: https://github.com/saltstack/salt/archive/v{{ version }}.tar.gz
sha256: 5cb7460741f44db445baf56e88acb5921e43dbefd5a8c24496f9b78718d96094
```